docs/weekly-questions/joes-water.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Joe's Water Corperation
+sidebar_position: 2
+---
+
+# Joe’s Doubling Points
+
+Joe's Water Corporation is a company that sells bottled water. However, he has competition that also sells bottled water, sometimes at a cheaper price.
+
+If your price is lower, then you get to sell at that store. If not, you lose the chance to sell your water at that store. In the case of a tie, the competition always wins.
+
+You, as the manager of Joe's Water Corporation, can choose to lower the price of water for every single store by M dollars. You should print out 'YES' if it makes you more revenue, or 'NO' if it makes you less revenue. (revenue = sum of prices at stores sold)
+
+### Input
+
+- Int K All the prices at joe's store
```

Comment:
```suggestion
- On the first line, a list of integers delimited by spaces, representing the prices at your stores.
```

Reply:
Why was this marked as resolved?

Reply:
cuz i want to collapse it so i can fix the next one